src/System.ComponentModel.TypeConverter/src/System/ComponentModel/BindableAttribute.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
+/*
+ */
```

Comment:
Nit: In addition to the leading whitespace, some files have these empty comments.